gradio/routes.py

```diff
@@ -1436,6 +1437,35 @@ def robots_txt():
             else:
                 return "User-agent: *\nDisallow: "
 
+        @app.get("/manifest.json")
+        def manifest_json():
+            if not blocks.pwa:
+                raise HTTPException(status_code=404)
+
+            return ORJSONResponse(
+                content={
+                    # NOTE: Required members: https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable#required_manifest_members
+                    "name": app.get_blocks().title,
```

Comment:
Instead of making the change above, let's do something like this 
```suggestion
                    "name": app.get_blocks().title or "Gradio",
```